src/controllers/controller.js

```diff
@@ -0,0 +1,62 @@
+import User from '../models/model';
+
+class UsersController {
+  constructor() {
+    this.users = [];
```

Comment:
If you are using classes, you need to move this ti Users Model instance, but it's ok for now - that will be changed in module 3

Reply:
Noted -- after reviewing the suggested structure, I'm going to rename this as a service, which makes me think about it holding model data differently. Will update in 3.